ClientApp/src/components/landingPage/Contact.tsx

```diff
@@ -15,9 +15,67 @@ import useStyles from "./Styles";
 import dummyImg from "./../../styling/img/dummy-image.jpg";
 import { Mail, Facebook, Instagram } from "@material-ui/icons";
 
-const Contact = () => {
+
+export interface ContactsData {
+  city: string,
+  contactPerson: string,
+  email: string,
+  facebook: string,
+  instagram: string,
+}
+
+interface Props {
+  contacts : ContactsData[]
+}
+
+
+const Contact : React.FC<Props> = ({contacts}) => {
   const classes = useStyles();
 
+console.log(contacts);
+
+const listContacts = contacts.map((contact, index) =>
```

Comment:
Could listContacts be renamed to Contacts or ContactList ?

This could also be put into the return, replacing {listContacts} at line 90.